### PR TITLE
Add some null checks to make it work with Blender-exported meshes and…

### PR DESCRIPTION
**Pull request by carolhmj**
… make it more resilient.

### CODE REVIEW

File: Tools/MakeIncremental/Program.cs [hunk 1]
```diff
@@ -211,7 +211,7 @@ static void ProcessSourceFile(string input, string dstPath)
                 }
 
                 dynamic scene;
-                var outputDir = Path.GetDirectoryName(dstPath);
+                var outputDir = (dstPath != null && dstPath != "") ? Path.GetDirectoryName(dstPath) : Path.GetDirectoryName(input);
```
Comment (bghgary) [inline Tools/MakeIncremental/Program.cs:214]:
```suggestion
                var outputDir = String.IsNullOrEmpty(dstPath) ? Path.GetDirectoryName(input) : Path.GetDirectoryName(dstPath);
```

**Review by bghgary — COMMENTED** [review]
I don't know this code that well. How do you test these changes?

File: Tools/MakeIncremental/Program.cs (hunk 1 above)
Comment (pandaGaume) [reply Tools/MakeIncremental/Program.cs:214]: @carolhmj could we resolve ?

**Comment by carolhmj** [comment]
@pandaGaume updated now, sorry I missed that!

**Comment by carolhmj** [comment]
@bghgary about the testing, I found the problem while trying to use it on the basic cube exported from Blender: https://forum.babylonjs.com/t/incremental-babylon-guide-for-dummies/28453/21?u=carolhmj

**Comment by bghgary** [comment]
> @bghgary about the testing, I found the problem while trying to use it on the basic cube exported from Blender: https://forum.babylonjs.com/t/incremental-babylon-guide-for-dummies/28453/21?u=carolhmj

Okay. I'm assuming both code paths of the `outputDir` was tested. I don't know this code well enough to determine if the code is correct.

**Comment by bghgary** [comment]
@carolhmj Do we still want this?

**Comment by carolhmj** [comment]
@bghgary hmmm I haven't seen anyone complaining about this so I guess it's not needed? We can reopen if needed.

**Comment by bghgary** [comment]
Okay, closing for now. Please reopen if needed.

**Pull request closed without merging**